packages/block-library/src/table/edit.js

```diff
@@ -1,13 +1,11 @@
 /**
  * External dependencies
  */
-
```

Comment:
I added these lines because that was @aduth's preference some time ago. I don't really mind, but it is not a doc block... So maybe it's better with the line, or just one asterisk at the start.

Reply:
If we want to change it, let's update in one batch all of them to match one style.